hybridse/src/node/node_manager.cc

```diff
@@ -443,19 +443,18 @@ UnaryExpr *NodeManager::MakeUnaryExprNode(ExprNode *left, FnOperator op) {
 SqlNode *NodeManager::MakeCreateTableNode(bool op_if_not_exist, const std::string &db_name,
                                           const std::string &table_name, SqlNodeList *column_desc_list,
                                           SqlNodeList *table_option_list) {
-    int replica_num = 1;
-    int partition_num = 1;
+    CreateTableOptions options;
     SqlNodeList partition_meta_list;
     if (nullptr != table_option_list) {
         for (auto node_ptr : table_option_list->GetList()) {
             if (nullptr != node_ptr) {
                 switch (node_ptr->GetType()) {
                     case kReplicaNum: {
-                        replica_num = dynamic_cast<ReplicaNumNode *>(node_ptr)->GetReplicaNum();
+                        options.replica_num = dynamic_cast<ReplicaNumNode *>(node_ptr)->GetReplicaNum();
```

Comment:
do not resolve these config here. Set `SqlNodeList *table_option_list` as `CreateStmt` member directly